eng/common/scripts/Package-Properties.ps1

```diff
@@ -76,81 +76,76 @@ function Get-PkgProperties
 {
     Param
     (
-        [Parameter(Mandatory = $true)]
         [string]$PackageName,
-        [Parameter(Mandatory = $true)]
+        [string]$ArtifactName,
         [string]$ServiceDirectory
     )
 
-    $pkgDirectoryPath = $null
-    $serviceDirectoryPath = Join-Path $RepoRoot "sdk" $ServiceDirectory
-    if (!(Test-Path $serviceDirectoryPath))
+    if (!$PackageName -and !$ArtifactName)
     {
-        LogError "Service Directory $ServiceDirectory does not exist"
+        LogError "You must specify either PackageName or ArtifactName Argument"
         return $null
     }
-
-    $directoriesPresent = Get-ChildItem $serviceDirectoryPath -Directory
-
-    foreach ($directory in $directoriesPresent)
+    
+    if ($ServiceDirectory)
     {
-        $pkgDirectoryPath = Join-Path $serviceDirectoryPath $directory.Name
-
-        if ($GetPackageInfoFromRepoFn -and (Test-Path "Function:$GetPackageInfoFromRepoFn"))
+        if (Test-Path $ServiceDirectory)
```

Comment:
Do we have cases where we are passing in a path instead of just a service name?

Reply:
That should not be the case but I started to think the argument names might be confusing in some cases and someone might think it refers to the path and I though I'll just take care of that.

Reply:
As we talked lets assume this is never a path but just a name so lets not do this code branch.